command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
Once we settle down on this compliance/certification issue, I'm going to update this line with “conformant” → “compliant” to match [this][1].

[1]: https://github.com/opencontainers/runtime-spec/tree/v1.0.0-rc1#open-container-runtime-specification